CITATION.cff

```diff
@@ -23,8 +23,8 @@ title: "echopype"
 doi: 10.5281/zenodo.3906999
 # but pointing to a specific release/version will
 # require having to update this file with each release
-version: 0.5.4
-date-released: 2021-10-01
+version: 0.6.4
+date-released: 2023-01-03
 url: "https://github.com/OSOceanAcoustics/echopype"
```

Comment:
Add use of `repository-code` to be more specific. Saw it first in xarray CITATION.cff and double checked at https://github.com/citation-file-format/citation-file-format/blob/main/README.md. The `url` can then point to the RTD site.
```suggestion
url: "https://echopype.readthedocs.io"
repository-code: "https://github.com/OSOceanAcoustics/echopype"
```